CHANGELOG.md

```diff
@@ -6,6 +6,72 @@ in this file.
 Detailed changes to the Solar Forecast Arbiter Core python library can be found
 in the core documentation's [what's new](https://solarforecastarbiter-core.readthedocs.io/en/latest/whatsnew/index.html) series.
 
+
+## [1.0beta5] - 2020-04-23
```

Comment:
```suggestion
## [1.0beta5] - 2020-04-24
```

Database migrations have to be done carefully, so tomorrow morning is better

Reply:
definitely